app/Notifications/Admin/AdminCreateUserNotification.php

```diff
@@ -43,9 +42,9 @@ public function toMail($notifiable)
     {
         return (new MailMessage)
                     ->subject('Welcome to PARCELHUB Warehouse system! '. $this->user->name)
-                    ->line('Your temporary password is ' . $this->password)
-                    ->line('Please click the button below to verify your account, we also recommend changing your password after you verify.')
-                    ->action('Verify', url('/users?name=' . $this->user->name));
+                    ->line('Your account has been created. ')
+                    ->line('Please click the button below to reset your password.')
+                    ->action('Reset', url('/oauth/reset?email=' . $this->user->email));
```

Comment:
Incorrect. Should point them to center url, not warehouse url. Click this link will not go to center to reset